notebooks/NDVI example.ipynb

```diff
@@ -69,6 +102,41 @@
     "process_graph_ndvi = json.loads(datacube_ndvi.to_json())[\"process_graph\"]"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "id": "acefd05c",
+   "metadata": {},
+   "source": [
+    "Convert process graph into an evalscript and print info about the created evalscript. The 'invalid_node_id' tells us up to which node the process graph could be converted. If it's \"None\", the whole process graph could be converted into an evalscript."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 6,
+   "id": "be7b0c35",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "[{'evalscript': <pg_to_evalscript.evalscript.Evalscript object at 0x000001E94E02EF70>, 'invalid_node_id': None}]\n"
+     ]
+    }
+   ],
+   "source": [
+    "subgraph_evalscripts = convert_from_process_graph(process_graph_ndvi)\n",
+    "print(subgraph_evalscripts)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "196c3db0",
+   "metadata": {},
+   "source": [
+    "Save evalscript as string and print redable version."
```

Comment:
Typo: readable

I wouldn't say we "save evalscript as string", we generate the evalscript string.

Reply:
Fixed (both)